test/integration/installExtension.test.ts

```diff
@@ -0,0 +1,22 @@
+import { clean, tmpdir } from "../utils/helpers"
+import { runCodeServerCommand } from "../utils/runCodeServerCommand"
+
+describe("--install-extension", () => {
+  const testName = "installExtension"
+  let tempDir: string
+  let setupFlags: string[]
+
+  beforeEach(async () => {
+    await clean(testName)
+    tempDir = await tmpdir(testName)
+    setupFlags = ["--extensions-dir", tempDir]
+  })
+  it("should install an extension", async () => {
+    const extName = "wesbos.theme-cobalt2"
+    await runCodeServerCommand([...setupFlags, "--install-extension", extName], {})
+    const { stdout } = await runCodeServerCommand([...setupFlags, "--list-extensions"], {
```

Comment:
After thinking about this more, I don't think we should use `--list-extensions` to verify `--install-extension` passed. Reasons:
1. we're essentially saying `--install-extension` relies on `--list-extensions` in order to pass, which doesn't mimic real-world usage
2. spawns a code-server instance which increases run time for this test

I think instead we should check the `tempDir` for the extension directory using `fs.promises`

Reply:
That makes sense to me.  The original intent of this test was actually just to make sure there were no native module errors and installing/listing extensions was the easiest way to trigger those errors.  I do not remember if install alone triggers the errors or if both were required to trigger the error so if we do add a check to `tempDir` we might want to keep both (maybe put the list part in a separate test).

Reply:
that sounds ideal to me! I'll do that then re-request a review